setup.py

```diff
@@ -589,12 +640,12 @@ def build_extensions(self):
 on_rtd = os.environ.get('READTHEDOCS') == 'True'
 
 
-
-REQ_TF_VERSION = LooseVersion("1.8.0")
+# this version compiles 3.8
```

Comment:
I guess you mean it's targetted at Python 3.8? I would either change or remove it.